pkg/networkservice/core/eventchannel/monitor_server.go

```diff
@@ -30,16 +30,17 @@ type monitorConnectionServer struct {
 	servers   []networkservice.MonitorConnection_MonitorConnectionsServer
 	selectors []*networkservice.MonitorScopeSelector
 	executor  serialize.Executor
+	connectCh chan int
 }
 
 // NewMonitorServer - returns a networkservice.MonitorConnectionServer
 //                    eventCh - when Send() is called on any of the NewMonitorConnection_MonitorConnectionsServers
 //                              returned by a call to MonitorConnections, it is inserted into eventCh
-func NewMonitorServer(eventCh <-chan *networkservice.ConnectionEvent) networkservice.MonitorConnectionServer {
+func NewMonitorServer(eventCh <-chan *networkservice.ConnectionEvent, monitorConnect chan int) networkservice.MonitorConnectionServer {
```

Comment:
Can we apply here grpc pattern `WithXYZ()` to make this parameter as optional? For example `WithMonitorConnectChannel(...)` ?